src/file_helpers.star

```diff
@@ -16,15 +15,14 @@ def get_toml_configuration_artifact(wakunode_name, same_toml_configuration):
 
     return artifact_id, file_name
 
-
-def generate_template_data(services):
+def generate_template_data(services, port_id):
```

Comment:
Nice usability extension actually. Now that it is changed, we could also change the variable `test` to something better (my fault), like `complete_template_data` or something like that, since `test` is... bad. I totally missed that when I pushed the code. Could you do this change?

Reply:
Done! Re-named to targets_payload